src/App.js

```diff
@@ -1,7 +1,27 @@
 import { availableRoutes } from './routes';
 
+import { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+
+const ScrollToTop = () => {
```

Comment:
You could extract scroll to top into a different file maybe into a folder named utils. Just a suggestion

Reply:
yeah i will do that